public/js/components/SourceSearch.js

```diff
@@ -22,6 +23,7 @@ function searchResults(sources) {
   }
 
   return sources.valueSeq()
+    .filter(source => !isPretty(source.toJS()))
```

Comment:
I think this could be turned into one filter `filter(source => !isPretty(source.toJS()) || source.get("url"))` don't have my computer up to test though.